appengine/endpoints-frameworks-v2/backend/pom.xml

```diff
@@ -154,6 +154,15 @@
                     </jvmFlags -->
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
We usually do this without adding the plugin.

```xml
    <properties>
        <maven.compiler.target>1.8</maven.compiler.target>
        <maven.compiler.source>1.8</maven.compiler.source>
    </properties>
```

Note - apparently we need to fix a bunch of files.